admin/formprocess.php

```diff
@@ -39,7 +39,7 @@
   SET series_id='$series_id', issue_number='$issue_number', story_name='$story_name', release_date='$release_date', plot='$plot', cover_image='images/$cover_image_file', original_purchase='$original_purchase', wikiUpdated=1
   WHERE comic_id='$comic_id'";
   if (mysqli_query ( $connection, $insert_comic_query )) {
-      $message = "Record updated successfully with the following information";
+      echo '<META http-equiv="refresh" content="0;URL=/comic.php?comic_id=' . $comic_id . '&m=5">';
```

Comment:
I've been meaning to move the form processing over to the form pages the way I did with addseries.php. No need to have form processing in it's own script.